Backpack/src/main/java/net/skyscanner/backpack/snackbar/SnackbarCustomisationUtils.kt

```diff
@@ -0,0 +1,51 @@
+package net.skyscanner.backpack.snackbar
```

Comment:
Can we move this to a `internal` package to keep it consistent to what we've been doing recently. Afaik all functions here are intended to be internal

Reply:
Agreed